TTGSnackbar/TTGSnackbar.swift

```diff
@@ -15,16 +15,24 @@ import Darwin
 /**
  Snackbar display duration types.
  
- - Short:   1 second
- - Middle:  3 seconds
- - Long:    5 seconds
+ - Short:       1 second
+ - Middle:      3 seconds
+ - Long:        5 seconds
+ - Expensify1: 15 seconds
+ - Expensify2: 20 seconds
+ - Expensify3: 25 seconds
+ - Expensify4: 30 seconds
  - Forever: Not dismiss automatically. Must be dismissed manually.
  */
 
 @objc public enum TTGSnackbarDuration: Int {
     case short = 1
     case middle = 3
     case long = 5
+    case expensify1 = 15
```

Comment:
If we're going to use numbers, can the number match the value?

`expensify15`
`expensify20`